tests/recall-behavior.sh

```
@@ -631,6 +648,437 @@ ok "PERF: 400KB pathological segment completes inside the wall bound" '[ "$RC" =
+// Delete ONLY nativeSearch's per-segment deadline check. The per-line check, the `stopped ||`
+// escape and evidence mode are all left exactly as they are.
+const fs = require('fs'); const f = process.argv[2];
+let s = fs.readFileSync(f, 'utf8');
+const find = '        if (Date.now() > deadline) { stopped = true; break; }\n        const lower = seg.text.toLowerCase();';
+if (s.split(find).length - 1 !== 1) { console.error('MUTATE493-SEG: expected exactly 1 anchor'); process.exit(3); }
+fs.writeFileSync(f, s.split(find).join('        const lower = seg.text.toLowerCase();'));
+M493S
+node "$WORK/mutate493-seg.js" "$M493MUT"; MRC=$?
+ok "SEGMENT-BUDGET-MR2 sabotage landed (the per-segment check is gone, the per-line check is NOT)" \
+   '[ "$MRC" = 0 ] && [ "$(grep -cF "if (Date.now() > deadline) { stopped = true; break; }" "$M493MUT")" -eq 1 ] && grep -qF "if (Date.now() > deadline) { stopped = true; break; }
+      let obj; try { obj = JSON.parse(line); }" "$M493MUT" && node --check "$M493MUT"'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)